app/views/orders/index.html.erb

```diff
@@ -1,12 +1,29 @@
 <h1>Your Orders:</h1>
 
+<div class="nav ms-auto w-100">
+  <% if params.has_value?('delivered') %>
```

Comment:
IMO this might bit a bit hard to read, understand and maintain. I'd think about changing the UI a bit or creating a helper that you could interpolate into `class:`. That helper should assign a CSS class based on passed to it params

Reply:
for now, i'm dropping highlighting of selected sort method